### DISCUSSION TITLE
Markdown is NOT a binary file, rollback changes and recover the functionality in the PULL REQUEST

### DISCUSSION THREAD

**Discussion by Scoppio** [community/community#15940] [Pull Requests]
The current way markdown file changes are shown in the Pull Request is very pretty, I like it a lot, the problem is that it has completely broken every single tool to comment, to suggest changes, and just about anything that is good in the pull review process, it now considers the Markdown file as a binary, only shows it as its "Richtext format" and in truth made it impossible for things like ADRs, wiki pages, documents, README files and many others to be manipulated, reviewed and corrected by anyone.

This does affect me personally and the company where I work at in a deep level. So please, let us deal with markdown files as we always have. Thank you.

**Comment by filipenf** [comment]
👍🏽  Comments on the rich diff is definitely a feature that github should have, especially for markdown

**Comment by JLesDev** [comment]
Do your own risk & compliance with the following information I'm about to give you but I found an extension that makes this possible. 

https://microsoftedge.microsoft.com/addons/detail/markdown-pr-%E2%80%94-markdown-pr/agomibenjlnikaldoddminkjbokfocgb

It works from my end on Microsoft Edge. There is a similar extension for Chrome. However, if you are using Chrome you may find you have bigger issues than commenting on markdown files.

Have a good day :+1:
